dev/tidb-in-kubernetes/deploy/prerequisites.md

```diff
@@ -71,6 +71,48 @@ If the above command shows that the swap column is all `0`, then swap is disable
 
 In addition, to permanently disable swaps, remove all the swap-related entries in `/etc/fstab`.
 
+After all the above configurations are made, check whether you have configured [SMP IRQ Affinity](https://cs.uwaterloo.ca/~brecht/servers/apic/SMP-affinity.txt) on the machine. This configuration is to assign the interrupt of each device to different CPUs to prevent all interrupts from being sent to the same CPU, avoiding the potential performance bottleneck. For the TiDB cluster, the rate at which the network card processes packages has a great impact on the throughput of the cluster. Use the following methods to assign the interrupt of a network card to a specific CPU, thus taking advantage of the multiple cores to increase cluster throughput.
```

Comment:
- Split it into two paras to make it easier to read.
- Avoid extra article "the".

```suggestion
After all the above configurations are made, check whether you have configured [SMP IRQ Affinity](https://cs.uwaterloo.ca/~brecht/servers/apic/SMP-affinity.txt) on the machine. This configuration is to assign the interrupt of each device to different CPUs to prevent all interrupts from being sent to the same CPU, avoiding the potential performance bottleneck. For the TiDB cluster, the rate at which the network card processes packages has a great impact on the throughput of the cluster.

Use the following methods to assign the interrupt of a network card to a specific CPU, thus taking advantage of multiple cores to increase cluster throughput.
```